### PR TITLE
MEN-3060 Worklfows and Artifact-creator-worker mongo url fix

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
please find details in https://tracker.mender.io/browse/MEN-3060?focusedCommentId=104251&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-104251

File: docker-compose.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ services:
             file: common.yml
             service: mender-base
         environment:
-            - WORKFLOWS_MONGO_URL=mongodb://mongo-workflows:27017
+            WORKFLOWS_MONGO_URL: mongodb://mender-mongo:27017
```
Comment (tranchitella) [inline docker-compose.yml]: As mentioned on slack, I believe the different syntax is not important:
https://docs.docker.com/compose/compose-file/
You can express env variables both ways.

File: docker-compose.yml [hunk 2]
```diff
@@ -115,7 +115,7 @@ services:
...
```
Comment (tranchitella) [inline docker-compose.yml]: Same here, obv.

**Comment by lluiscampos** [comment]
Superseded by #793

**Comment by merlin-northern** [comment]
@tranchitella please, could we merge it, I need to check one thing, and it would help a lot.

**Comment by merlin-northern** [comment]
this one fixed the tests, see, e.g.: https://github.com/mendersoftware/mender-docs/pull/856

**Pull request merged**